src/rust/ide/src/tests.rs

```diff
@@ -42,6 +42,13 @@ async fn failure_to_open_project_is_reported() {
         }]
     }));
     fixture.when_stalled_send_error(1,"Service error");
+    // FIXME [mwu]
+    //  For some reasons, the line below is needed, even though dropping the fixture does the very
+    //  same call. Apparently `run_until_stalled` must be called twice. This looks like an error,
+    //  and happens only for wasm targets.
+    //  This did not happen on the 2019 nightly.
+    //  Further investigation needed.
```

Comment:
"the line below is needed" - the comment lacks the most important part - what happens without this line? Please add it there so the comment explains the whole world

Reply:
Comment has been updated.